### PR TITLE
Load resizable jquery-ui chunk for knockout resizable binding only when actually used

### LINKED ISSUE #32753: Resizable knockout binding does not work on the frontend
### Preconditions (*)

1. Magento 2.4.2

### Steps to reproduce (*)

1. Add resizable knockout binding for UI Component template (for example minicart) as described in the documentation: https://devdocs.magento.com/guides/v2.4/ui_comp_guide/concepts/knockout-bindings.html#resizable

### Expected result (*)

1. There is a resizable element visible, with controls attached
2. No error in the console

### Actual result (*)

1. There is a console error: 
error
2. if line 124 in resizable.js is commented out resizable is binded without error. However as styles are not included controls that are added are not visible, so component is useless.
resizab

It seems to me that this component is used only in the admin panel. I have checked issues and no one reported problems with resizable. Maybe it would be useful to mark this binding as admin only and do not load unnecessary javascript on the frontend side.

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.4/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [x] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

### PR DESCRIPTION
**Pull request by martasiewierska**
### Description (*)

Resizable knockout binding is hardly never used on the frontend, it also does not work there (see https://github.com/magento/magento2/issues/32753) As we are constantly often struggling to reduce unused javascript and improve performance metrics for our shops I would like to add this small step to remove not used javascript.

### Manual testing scenarios (*)

1. Create empty page without any other component that require _jquery/ui-modules/resizable.js_
2. Check if _jquery/ui-modules/resizable.js_ file is not loaded in Network tab (no js bundling)
3. Create test UIComponent with resizable knockout binding
4. Check if _jquery/ui-modules/resizable.js_ file is loaded in Network tab (no js bundling)

### Fixed Issues (if relevant)

1. Fixes magento/magento2#32753

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ptylek** [comment]
Looks good to me. Let's wait for test results.
@magento run all tests

**Comment by mrtuvn** [comment]
@magento run Functional Tests B2B,  Functional Tests CE, Functional Tests EE

File: app/code/Magento/Ui/view/base/web/js/lib/knockout/bindings/resizable.js [hunk 1]
```diff
@@ -139,7 +138,11 @@ define([
         init: function (element, valueAccessor, allBindings, viewModel) {
             var config = processConfig(valueAccessor(), viewModel, element);
 
-            $(element).resizable(config);
+            require(['jquery-ui-modules/resizable'], function () {
+                if($.fn.resizable) {
```
Comment (ihor-sviziev) [inline app/code/Magento/Ui/view/base/web/js/lib/knockout/bindings/resizable.js]:
```suggestion
                if ($.fn.resizable) {
```

**Comment by ihor-sviziev** [comment]
@mrtuvn, I believe it would be too hard to cover these changes with any test. I prefer not to cover it. What do you think?

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by mrtuvn** [comment]
ihor Ok let's see final test

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Integration Tests, WebAPI Tests

**Comment by ptylek** [comment]
@magento run Functional Tests B2B, Functional Tests EE, WebAPI Tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Review by ihor-sviziev — APPROVED** [review]

**Review by ptylek — APPROVED** [review]

**Comment by dmitriyprime** [comment]
:heavy_check_mark:  QA passed

**Comment by ihor-sviziev** [comment]
Moving this PR back to "accept" status...

**Pull request merged**